public/css/styles.css

```diff
@@ -154,10 +154,12 @@ a.btn.btn-light {
 .search-term {
   text-align: center;
   margin-top: 2rem;
+  margin-bottom: 0;
+  font-size: 1rem;
 }
 
 .card-deck {
-  padding: 2rem;
+  /* padding: 0 2rem 2rem 2rem; */
```

Comment:
how come you removed padding on top? does it look good even when search term text is missing?

Reply:
Just tested locally and it looks a bit imbalanced. How about instead of removing padding on the card deck, you put a negative margin on search text, like this?

```
search-term {
    margin-bottom: -2rem;
}
```